src/Components/Components/src/RenderTree/RenderTreeFrame.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.AspNetCore.Components.RenderTree
     /// <summary>
     /// Represents an entry in a tree of user interface (UI) items.
     /// </summary>
-    [StructLayout(LayoutKind.Explicit)]
+    [StructLayout(LayoutKind.Explicit, Pack = 4)]
```

Comment:
`Pack = 4` is now needed because by default, Mono picks a struct length that's the smallest multiple of the largest field size. The largest field is now 8 bytes, so it would pick struct length 40. Explicitly specifying `Pack=4` lets it pick a multiple of 4, so with this it picks struct length 36.

The existing TS-side code already expects 36. We could change it to 40 but with this line, there's no need.